packages/legend-query/src/components/QueryBuilderMilestoneEditor.tsx

```diff
@@ -206,20 +204,20 @@ const BiTemporalMilestoneEditor = observer(
       <>
         <div className="panel__content__form__section">
           <div className="panel__content__form__section__header__label">
-            Business Date
+            Processing Date
           </div>
           <MilestoningParameterEditor
-            key="BusinessDate"
+            key="ProcessingDate"
             queryBuilderState={queryBuilderState}
             index={0}
           />
         </div>
         <div className="panel__content__form__section">
           <div className="panel__content__form__section__header__label">
-            Processing Date
+            Business Date
           </div>
           <MilestoningParameterEditor
-            key="ProcessingDate"
+            key="BusinessDate"
             queryBuilderState={queryBuilderState}
             index={1}
```

Comment:
Could we spend a little bit of time to rename this to `parameterIndex` or something? `index` is kinda vague